### PR TITLE
Add more details about options to create_merge_request doc

### PR DESCRIPTION
**Pull request by fabioperrella**
To avoid issues like the https://github.com/NARKOZ/gitlab/issues/323 this MR aims to improve the doc of create_merge_request command.

#hacktoberfest

### CODE REVIEW

**Comment by timofonic** [comment]
@fabioperrella Nice!
@NARKOZ What do you mean by Hacktoberfest? Can this be merged? :D

**Comment by fabioperrella** [comment]
@timofonic -> https://hacktoberfest.digitalocean.com/

File: lib/gitlab/client/merge_requests.rb [hunk 1]
```diff
@@ -44,6 +44,7 @@ def merge_request(project, id)
     # @option options [String] :target_branch (required) The target branch name.
     # @option options [Integer] :assignee_id (optional) The ID of a user to assign merge request.
     # @option options [Integer] :target_project_id (optional) The target project ID.
+    # @option options [String] :labels (optional) Labels as a comma-separated list. To see all available options, go to https://docs.gitlab.com/ee/api/merge_requests.html#create-mr
```
[the remark this answers is not in the document]
Comment (fabioperrella) [reply lib/gitlab/client/merge_requests.rb]: done!

**Comment by NARKOZ** [comment]
Thank you.

**Pull request merged**